01_deploy_nodes.py

```diff
@@ -133,6 +134,29 @@ def run(self):
             pass
         os.symlink(self.result_dir, link)
 
+        ## create ssh tunnels file
+        logger.info("ssh tunnel informations:")
+        logger.info("___")
+        
+        script = "cat > /tmp/openstack_ssh_config <<EOF\n"
+        script += "Host *.grid5000.fr\n"
+        script += "  User "+self.user+" \n" 
+        script += "  ProxyCommand ssh -q "+self.user+"@194.254.60.4 nc -w1 %h %p # Access South\n"
+        script += "EOF\n"
+        
+        port = 8080
+        for host in roles['controllers']:
+            port = port + 1
+            script += "ssh -F /tmp/openstack_ssh_config -N -L "+`port`+":"+host.address+":80 "+self.user+"@frontend."+get_host_site(host)+".grid5000.fr &\n" 
```

Comment:
I guess we could remove the dependency on  `execo_g5k.api_utils` by using `self.user@access.grid5000.fr` directly